raft/node.go

```diff
@@ -499,6 +502,13 @@ func (n *node) ReportSnapshot(id uint64, status SnapshotStatus) {
 	}
 }
 
+func (n *node) TransferLeadership(ctx context.Context, lead, transferee uint64) {
+	select {
+	case n.recvc <- pb.Message{Type: pb.MsgTransferLeader, From: transferee, To: lead}:
```

Comment:
it is fine in that case. Usually it is the transferee to ask the leader to give the leadership to it. So from is transferee and to is the leader. We are on the opposite that the leader now gives the leadership to a follower voluntarily. So we need to doc this.